.github/workflows/review-code.yaml

```
@@ -0,0 +1,53 @@
+    types:
+      - opened
+      - synchronize
+      - reopened
+
+jobs:
+  reviewmycode:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout PR Code
+        uses: actions/checkout@v2
```

actionlint

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)